elements/rh-band/src/rh-band.html

```diff
@@ -0,0 +1,15 @@
+<div class="rh-band__wrapper">
+  <div class="rh-band__container">
```

Comment:
I might be telling you something you already know, but in webrh we solved the left/right side issue with the following:
```
{# use flexbox to reverse row #}
{% set flex_swap = null %}
{% if (desktop_aside == "right" and aside_mobile == "top") or (desktop_aside == "left" and aside_mobile == "bottom") %}
    {% set flex_swap = "swap" %}
{% endif %}
```

so for this web component, this might be all you need:
```
:host([aside*="right"][aside*="top"]) .rh-band__container,  
:host([aside*="left"][aside*="bottom"]) .rh-band__container  {
    @media (min-width: #{rh-breakpoint(md)}) {
      flex-direction: row-reverse;
    }
  }
```

Note i tried combining these like this `[aside*="right top"]` to no avail

Reply:
Fixed this with a really amazing grid layout!